replication-api-impl/src/main/java/org/codice/ditto/replication/api/impl/persistence/ReplicationPersistentStore.java

```diff
@@ -98,7 +106,10 @@ public ReplicationPersistentStore(PersistentStore persistentStore) {
           "An error occurred while retrieving data of type " + typeName, e);
     }
 
-    return storedMaps.stream().map(map -> getPersistableFromMap(klass, map));
+    return storedMaps
```

Comment:
✏️ no need for this change if we let the `fromMap()` take care of incompatibilities in versions.